.travis.yml

```diff
@@ -0,0 +1,26 @@
+sudo: false
+
+language: php
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
+php:
+  - 5.6
+  - 7.0
+  
+matrix:
+  fast_finish: true
+  include:
+    - php: 7.0
+      env: deps=low
+
+before_script:
+  - phpenv config-rm xdebug.ini
+  - composer self-update
+  - if [[ "$deps" = "low" ]]; then composer update --prefer-lowest --prefer-stable; else composer install --prefer-dist; fi
```

Comment:
Oh well, I didn't want to break your commit (https://github.com/xabbuh/ParamConverterBundle/commit/6c10ce6da7a2afa4c01f72e010a51e12e57283fa#diff-354f30a63fb0907d4ad57269548329e3R9). ;) No, you got me with this. :D